src/main/java/com/xjeffrose/xio/SSL/X509CertificateGenerator.java

```diff
@@ -58,14 +58,14 @@ public static DERKeySpec parseDERKeySpec(Path path) {
     return parseDERKeySpec(rawKeyString);
   }
 
-    public static DERKeySpec parseDERKeySpec(String rawKeyString) {
+  public static DERKeySpec parseDERKeySpec(String rawKeyString) {
     try {
       // Base64 decode the data
       Base64.Decoder b64decoder = Base64.getDecoder();
       byte[] encoded = b64decoder.decode(
-          rawKeyString.replace("-----BEGIN RSA PRIVATE KEY-----\n", "")
-              .replace("-----END RSA PRIVATE KEY-----\n", "")
-              .replace("\n", "")
+        rawKeyString.replace("-----BEGIN RSA PRIVATE KEY-----", "")
+          .replace("-----END RSA PRIVATE KEY-----", "")
+          .replace(System.getProperty("line.separator"), "")
```

Comment:
Personally I don't think so -- as long as the PEM file is treated as text and standard git practices are followed this would be the portable solution.  You suggestion would work and may be safer but i still feel unncessary